spec/support/lti_controller_examples.rb

```diff
@@ -0,0 +1,136 @@
+shared_examples 'lti deployment controller' do
+  let(:instructor) { create :instructor }
+  let!(:client_id) { 'LMS defined ID' }
+  let(:target_link_uri) { 'https://example.com/authorize_redirect' }
+  let(:host) { 'https://canvas.instructure.com' }
```

Comment:
if this is a shared example for all lti controller subclasses, do not use canvas specific examples